libs/pageserver_api/src/key.rs

```diff
@@ -522,6 +522,30 @@ pub fn is_slru_block_key(key: Key) -> bool {
         && key.field6 != 0xffffffff // and not SlruSegSize
 }
 
+#[inline(always)]
+pub fn key_range_size(key_range: &Range<Key>) -> u32 {
+    let start = key_range.start;
```

Comment:
This function already exists in `libs/pageserver_api/src/keyspace.rs`

Reply:
Yeah I copied/moved it to key.rs  in my rebase on dec 1st's state on main, where Key has been moved to pageserver_api, but that function hasn't been (yet). then a later commit on main did the move but it moved it to the file you specify. I missed that part, will correct it.

Reply:
removed the function.